CMakeLists.txt

```diff
@@ -220,6 +220,12 @@ IF(WIN32)
     ENDIF()
 ENDIF(WIN32)
 
+# pkg-config support
+find_package(PkgConfig)
+IF(NOT PKG_CONFIG_FOUND)
+    MESSAGE(STATUS "Will not provide a pkg-config file.")
+ENDIF(NOT PKG_CONFIG_FOUND)
+
```

Comment:
Shouldn't this be guarded to prevent it from being run on OSes without pkg-config support like Windows?